meetings/2024-06-26.md

```diff
@@ -0,0 +1,105 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2024-06-26
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=DlrCDjGPKgQ>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1585>
+
+## Present
+
+* Antoine du Hamel @aduh95 (voting member)
+* Benjamin Gruenbaum @benjamingr (voting member)
+* Ruben Bridgewater @BridgeAR (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Marco Ippolito @marco-ippolito (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Richard Lau @richardlau (voting member)
+* Robert Nagy @ronag (voting member)
+* Ruy Adorno @ruyadorno (voting member)
+* Paolo Insogna @ShogunPanda (voting member)
+* Tobias Nießen @tniessen (voting member)
+* Robin Ginn (Guest - OpenJS Executive director)
+
+## Agenda
+
+### Announcements
+
+* Rafael: Heads up - security release planned for July 2nd.
+
+### Reminders
+
+* Remember to nominate people for the [contributor spotlight](https://github.com/nodejs/node/blob/main/doc/contributing/reconizing-contributors.md#bi-monthly-contributor-spotlight)
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No update this week, but board meeting is this Friday.
+
+### nodejs/node
+
+* doc: deprecate NaN or negative number as delay to setTimeout and setInterval [#53005](https://github.com/nodejs/node/pull/53005)
+  * Benjamin - believe we have achieved consensus. Going to change process to allow warnings
+    Outside of deprecations. Wanted warning if something that gets accidentally converted
+    To 0.
```

Comment:
```suggestion
    To 1.
```

Reply:
I said 0 in the meeting but this is technically correct